integration-tests/src/docker/build.ts

```diff
@@ -3,12 +3,17 @@ import * as core from '@actions/core'
 import simpleGit from 'simple-git'
 
 import {runShellCommands} from '../utils/system'
+import path from "path";
+import fs from "fs";
 
 export async function buildDockerTestImage(
+  testsFolder: string,
```

Comment:
Aren't `testsFolder` and `testsPath` serving the same purpose ? 

I suspect you don't need to use both here. If you need `testsFolder` in order to run the ci.build.sh script, I believe it will work as well for the `docker save` command, in that case you can remove `testsPath` from the function.

Reply:
`testFolder` is equivalent to `rootProjectFolder/tests_path`
The docker commands use the relative test path and I was wondering if I could use it to check if a build script exists